src/Domain/Creators/SphereTimeDependentMaps.cpp

```diff
@@ -145,10 +167,10 @@ TimeDependentMapOptions::MapType<Frame::Grid, Frame::Inertial>
 TimeDependentMapOptions::grid_to_inertial_map(
     const bool include_distorted_map) const {
   if (include_distorted_map) {
-    return std::make_unique<GridToInertialComposition>(shape_map_);
+    return std::make_unique<GridToInertialComposition>(shape_map_,
+                                                       translation_map_);
   } else {
-    return std::make_unique<
-        IdentityForComposition<Frame::Grid, Frame::Inertial>>(IdentityMap{});
+    return std::make_unique<GridToInertialSimple>(translation_map_);
```

Comment:
Take another bool like `transition_translation_map` to determine if this block should use the transition or the rigid translation.